back/app/services/idea_custom_fields_service.rb

```diff
@@ -175,16 +170,19 @@ def duplicate_all_fields
 
   private
 
-  # Replace a point field with two fields, one for latitude and one for longitude,
-  # so that the XlsxExport::InputSheetGenerator and BulkImportIdeas::IdeaXlsxFormExporter#export
-  # can produce separate columns for latitude and longitude.
-  def replace_point_fields_with_lat_and_lon_point_fields(fields)
+  def add_suffix_to_geo_fields_title_multiloc(fields)
     fields.map do |field|
-      if field.input_type == 'point'
-        [field.point_latitude_field, field.point_longitude_field]
-      else
-        field
+      if CustomField::GEOGRAPHIC_INPUT_TYPES.include? field.input_type
+        field.title_multiloc = field.title_multiloc.to_h do |k, v|
```

Comment:
Maybe personal preference, but find the code more readable if `k` & `v` would be something more specific to the hash. In this case something like `locale` & `title`

Reply:
Good point. Also my preference, so I must have been feeling lazy. Will change.